pkg/tenant/tenant.go

```diff
@@ -66,6 +67,8 @@ type Server struct {
 	quarterdeck qd.QuarterdeckClient // client to issue requests to Quarterdeck
 	sendgrid    *emails.EmailManager // send emails and manage contacts
 	tasks       *tasks.TaskManager   // task manager for performing background tasks
+	topics      *TopicSubscriber     // consume topic updates from Ensign
+	wg          *sync.WaitGroup      // waitgroup for coroutines
```

Comment:
technically these are go routines (system threads) not coroutines (which operate in the same thread) … guess you've been spending a lot of time in the Python async SDK! ;-)